dev-libs/ocl-icd/ocl-icd-2.2.12-r2.ebuild

```diff
@@ -26,7 +26,7 @@ src_prepare() {
 }
 
 multilib_src_configure() {
-	ECONF_SOURCE="${S}" econf --enable-pthread-once
+	ECONF_SOURCE="${S}" econf --enable-pthread-once $(use_enable khronos-headers official-khronos-headers)
```

Comment:
Split long lines.

Generally, you want to aim for no more than 80 characters on a line, but don't bend over backwards to do so.

Reply:
No problem. I refrained from splitting it for the sake of consistency as there still was a longer line below.